pkg/plugin/interfaces.go

```diff
@@ -119,8 +139,9 @@ type EditSubcommand interface {
 
 // Full is an interface for plugins that provide `init`, `create api`, `create webhook` and `edit` subcommands
 type Full interface {
-	Init
-	CreateAPI
-	CreateWebhook
-	Edit
+	Plugin
+	InitGetter
+	CreateAPIGetter
+	CreateWebhookGetter
+	EditGetter
```

Comment:
I'm not sure what changed but this was giving me duplicate methods error as all of the wrapped interfaces were wrapping `Plugin` themselves. So I basically made interfaces that didn't wrap `Plugin` as an intermediate step.

Reply:
I'm not seeing any issues even after embedding `Plugin` in `Full`.

Reply:
Strange, go version? I found this error in #1869 too when I run `go test`, I think I'm using 1.15

Reply:
That's strange. Composite interfaces with duplicate methods were [added](https://medium.com/@meeusdylan/changes-to-interfaces-in-go1-14-821ab533f62) in go v1.14.

Reply:
Will revert this change